components/pagerduty/actions/find-oncall-user/find-oncall-user.mjs

```diff
@@ -0,0 +1,47 @@
+import pagerduty from "../../pagerduty.app.mjs";
+
+export default {
+  key: "pagerduty-find-oncall-user",
+  name: "Find Oncall User",
+  description: "Find the user on call for a specific schedule. [See the docs here](https://developer.pagerduty.com/api-reference/b3A6Mjc0ODE0MA-create-an-incident)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    pagerduty,
+    scheduleId: {
+      propDefinition: [
+        pagerduty,
+        "oncallScheduleId",
+      ],
+    },
+    userId: {
+      optional: false,
```

Comment:
Since this prop is in general probably required, would it be better to remove the `optional: true` from the app and also remove this line? See the following comment: https://github.com/PipedreamHQ/pipedream/pull/2013#discussion_r779210415

```suggestion
```